ignite/cmd/node_query_tx.go

```diff
@@ -24,11 +23,12 @@ func nodeQueryTxHandler(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
-	rpc, err := sdkclient.NewClientFromNode(getRPC(cmd))
+	client, err := newNodeCosmosClient(cmd)
 	if err != nil {
 		return err
 	}
-	resp, err := rpc.Tx(cmd.Context(), bz, false)
+
+	resp, err := client.RPC.Tx(cmd.Context(), bz, false)
 	if err != nil {
 		return err
```

Comment:
```suggestion
		return rpcError(c.nodeAddress, err)
```

Should we wrap this error?

Reply:
No because `client.RPC` is already the wrapped rpc.

That's actually the reason of this change in the `node query tx` command: using `cosmosclient` instead of the RPC client directly ensures the errors are wrapped.